pkg/gardenlet/controller/shoot/shoot.go

```diff
@@ -192,8 +192,9 @@ func (c *Controller) Run(ctx context.Context, shootWorkers, shootCareWorkers, sh
 		// Check if the status indicates that an operation is processing and mark it as "aborted".
 		if shoot.Status.LastOperation != nil && shoot.Status.LastOperation.State == gardencorev1beta1.LastOperationStateProcessing {
 			newShoot := shoot.DeepCopy()
+			patch := client.MergeFrom(newShoot)
```

Comment:
Shouldn't this be

```suggestion
			patch := client.MergeFrom(newShoot.DeepCopy())
```

?

Reply:
`newShoot` is already Deepcopy.